Library/Homebrew/software_spec.rb

```diff
@@ -385,9 +385,9 @@ def tab_attributes
     manifests_annotations = manifests.map { |m| m["annotations"] }
     raise ArgumentError, "Missing 'annotations' section." if manifests_annotations.blank?
 
-    bottle_digest = @resource.checksum.hexdigest
+    image_ref = GitHubPackages.version_rebuild(@resource.version, rebuild, @tag.to_s)
     manifest_annotations = manifests_annotations.find do |m|
-      m["sh.brew.bottle.digest"] == bottle_digest
+      m["org.opencontainers.image.ref.name"] == image_ref
```

Comment:
```suggestion
      next if m["sh.brew.bottle.digest"] != bottle_digest
 
      m["org.opencontainers.image.ref.name"] == image_ref 
```

would like to still keep the checksum check here